ndmg/stats/qa_reg.py

```diff
@@ -121,7 +136,7 @@ def plot_brain(brain, minthr=2, maxthr=95, edge=False):
         for pos in coord:
             idx += 1
             ax = fbr.add_subplot(3, 3, idx)
-            ax.set_axis_bgcolor("black")
+            ax.set_axis_bgcolor("white")
```

Comment:
another example of code I've been seeing repeated. Try to consolidate with the other PRs.